### ISSUE TITLE
Option to Support schema.org SoftwareSourceCode subset alone

### ISSUE THREAD

**Issue by cboettig** [ropensci/codemetar#47]
Some projects / applications may wish to use only terms supported in schema.org SoftwareSourceCode type.  Consider a utility that can generate just this codemeta information.

**Comment by cboettig** [comment]
yes; should set the context to just "http://schema.org", and at least the dropping terms only in the codemeta namespace and not in schema.org.  Technically should probably drop those terms that are part of `SoftwareApplication` and not part of SoftwareSourceCode, but they aren't invalid in the schema context so they could be left in.  That is, this basically happens automatically if you just `jsonld_compact` into "http://schema.org" namespace....

Still, any reasonably-engineered utility that can parse schema.org:SoftwareSourceCode should just be doing this compaction automatically anyway, so doing this manually is arguably silly and maybe we should just close this issue?

**Comment by maelle** [comment]
as you think it's best!
